tritonclient-script.py

```diff
@@ -47,8 +47,8 @@ def triton_predict_grpc(triton_client, input_arr, model_name, batch_size=1, mode
     # caution: if the connection to localhost does not work,
     # use container IPAddres obtained e.g. from:
     # docker inspect triton | grep IPAddress
-    # triton_host = 'localhost'
-    triton_host = '172.17.0.2'
+    triton_host = 'localhost'
```

Comment:
Notice that `localhost` is not shared between two containerized systems, hence the need for the use of ephemeral "external" container IP addresses. Client and server containers should probably be run with `docker compose` and share the same `docker network`).